website/content/en/docs/golang/kubebuilder/crds-scope.md

```diff
@@ -91,5 +97,5 @@ spec:
 ``` 
   
 [RBAC]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
-[manager_user_guide]: /docs/golang/quickstart/#manager
+[manager_user_guide]:/docs/golang/kubebuilder/quickstart/#manager
```

Comment:
Should we keep it as kubebulder? 
Should not better we use other nomenclature as new layout for example?

Reply:
If I understand what @camilamacedo86 is suggesting correctly, I _think_ I agree.

Could we do the following?
```
git mv website/content/en/docs/golang/ website/content/en/docs/golang-legacy
git mv website/content/en/docs/kubebuilder/ website/content/en/docs/golang/
```

Any downsides I'm not thinking of?

Reply:
@joelanford @camilamacedo86 
I was trying to go for the same weight between the two Go operator docs sections i.e:
```
- Golang Operator
  - New CLI docs
    - user-guide
    - ...
  - Legacy CLI docs
    - user-guide
    - ...
```

Nesting the legacy docs as a subsection within the old docs would look like:
```
- Golang Operator New CLI docs
  - user-guide
  - ....
  - Legacy CLI docs
    - ...
```

If that seems okay docs UX wise, then I can make the update.

Also @asmacdo any thoughts on how the link tree should look like for legacy and new docs?

Reply:
I prefer the second option. Shows more clear. 👍

Reply:
I like option 2 as well. It seems like it would put the focus on the new docs.

Reply:
@joelanford @camilamacedo86 Done:

![image](https://user-images.githubusercontent.com/6660644/84217547-4c8c7d00-aa81-11ea-8111-eaa312c39235.png)


However now that I've: 
-  moved `docs/kubebuilder` to `docs/golang`, 
- and nested `docs/golang` at `docs/golang/legacy`

the git diff thinks I'm rewriting `website/content/en/docs/golang` instead of simply renaming it.

So the diffs for both docs are going to be larger than what they really are, which makes this kind of a pain to review.

One alternative is for me not to nest the docs in this PR and just rename them to separate locations so it's easier to review:
- `docs/golang/new`
- `docs/golang/legacy`

And I can reshuffle them back in a follow up PR after we're okay with the actual contents and have them merged.

Or if you're fine reviewing this, then this can stay.

Reply:
@joelanford @camilamacedo86 I've reshuffled the docs (again) to avoid having `git diff` show renaming the new docs as rewriting the old ones.

So I would suggest that we keep the current legacy/new doc paths and linktree layout as a placeholder, and not nest the legacy docs inside the new ones for this PR.

And once the actual doc contents are reviewed and this PR gets merged, I can reshuffle the docs again to the nested layout structure from above.

For now it's going to be:
![image](https://user-images.githubusercontent.com/6660644/84610603-fe57ef00-ae6f-11ea-9608-7a97de19bd7f.png)

Same goes for the actual titles of the links. We can refine the names later.